point/이_거니와.yaml

```
@@ -0,0 +1,48 @@
+name: (이)거니(와)
```

```suggestion
name: (이)거니
```

I would go without the 와, it's just another particle and it's related to only one of the definition (the definition with 와 is great and should be there)